crates/utils/src/error.rs

```diff
@@ -179,6 +179,8 @@ pub enum LemmyErrorType {
   UrlWithoutDomain,
   InboxTimeout,
   Unknown(String),
+  NotAnActor,
+  CantDeleteSite,
```

Comment:
Adding an error here means that it needs to get translated into all different languages and included by clients. However the translations will never be used. So it would be good to create another error enum only for federation errors.